recipe/meta.yaml

```diff
@@ -38,8 +38,8 @@ source:
       # work-around for https://github.com/grpc/grpc/issues/36654
       - patches/0010-set-Wl-undefined-dynamic_lookup-for-upb_-json-textfo.patch
       - patches/0011-add-missing-GRPC_DLL-macro.patch
-  - url: https://pypi.io/packages/source/g/grpcio-tools/grpcio_tools-{{ version }}.tar.gz
-    sha256: 7c3a47ad0070bc907c7818caf55aa1948e9282d24e27afd21015872a25594bc7
+  - url: https://pypi.io/packages/source/g/grpcio-tools/grpcio_tools-{{ version }}rc1.tar.gz
+    sha256: 390aaf8c28036c0319a4af8c861815a8adafc8110bea0160a9a5341e0f520523
```

Comment:
Quoting a comment from https://github.com/conda-forge/libprotobuf-feedstock/pull/237 for viz:
> > @xhochy: Sadly(?) `grpcio-tools` has only published `rc1` yet.
> 
> @h-vetinari: We could build `grpcio-tools` from source -- it's the same repo anyway; when I did [conda-forge/grpc-cpp-feedstock#370](https://github.com/conda-forge/grpc-cpp-feedstock/pull/370) I just wanted to transpose the tools-feedstock and unblock grpc, not dive down yet another rabbit hole. But if it ends up not too hard, we'd spare ourselves the dependency on waiting for pypi releases.

Reply:
I'm split here. For one, the code is part of the official tag. On the other hand, it seems that for 1.66.0, they went through several release candidates (but they were also reflected as tags 🤔 ).

For now, I would focus first on getting tensorflow passing the current migration and if that works/is finished, I would attempt to build from the main source tree.

Reply:
The official sdist has been published, I'll update and merge.